problems/4.arrays.js

```diff
@@ -2,27 +2,38 @@
 * @param {number[]} arr - The input array
 * @returns {number} - The average of the first and last values
 */
-function firstAndLastAverage() {
-
+function firstAndLastAverage(arr) {
+  let arr = []
+  for (i = 0; i < arr.length; i++){
+    return (arr[0] / arr[2])
+  }
+  return arr
 }
 
+
 /** Duplicates all values inside an array.
 * Sample input: [1,4,2,5]
 * Sample output: [1,4,2,5,1,4,2,5]
 * @param {number[]} arr - The input array
 * @returns {number[]} - The duplicated array
 */
-function duplicateArray() {
-
+function duplicateArray(arr) {
+let oldArr = [1,4, 2, 5];
```

Comment:
Stop making up what values your variables have in the function. Input is what will be given to you. Think about he parameter as something you'll have by subject. A number, an array. NOT what number, what array.